github-review.el

```diff
@@ -126,6 +126,16 @@ return a deferred object"
     (let ((query (format "query {
   repository(name: \"%s\", owner: \"%s\") {
     pullRequest(number: %s){
+      reviewThreads(first: 100) { edges { node {comments(first: 10) {
```

Comment:
Starts the 2nd view.

Reply:
A reply to 2.0 as solitary comment over the 1-2  commit range patch. Expected `originalCommitId = HEAD`.

Reply:
A solitary comment review reply via thread to find out `original-commit-id`.